package.json

```diff
@@ -61,7 +61,7 @@
         "webpack-remove-empty-scripts": "^1.0.3"
     },
     "dependencies": {
-        "autoql-fe-utils": "1.0.14",
+        "autoql-fe-utils": "^1.0.15",
```

Comment:
Can we change this to the exact version instead of using the caret? I think locking the exact version is safer for the util library in case there are minor changes we dont want to include by accident. 

You can do it by running `npm i autoql-fe-utils@latest --save-exact`